scripts/fetch-asyncapi-example.js

```diff
@@ -82,11 +82,11 @@ const buildCLIListFromExamples = async () => {
       };
     } catch (error) {
       // Failed for somereason to parse this spec file, ignore for now
-      console.error(error);
+      return null;
     }
   });
 
-  const exampleList = await Promise.all(buildExampleList);
+  const exampleList = (await Promise.all(buildExampleList)).filter(item => item !== null);
```

Comment:
Using double negation would make this compatible with any scenario. Then you won't need to change the return [here](https://github.com/asyncapi/cli/pull/1033/files#diff-3796d0c36710ab426831cd95695d7165c7fe53517159316d107a3678bdb4705fR74) to be explicitly a `null` 
```suggestion
  const exampleList = (await Promise.all(buildExampleList)).filter(item => !!item);
```

Does it make sense? WDYT?